<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyRecord</class>
 <widget class="QDialog" name="modifyRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(7, 7, 7);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="projectFrame">
        <property name="styleSheet">
         <string notr="true">color: rgb(186, 186, 186);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="projectLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Project: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="projectList">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Project1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Project2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Project3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Project4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Project5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Project6</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>25</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="startFrame">
        <property name="styleSheet">
         <string notr="true">color: rgb(186, 186, 186);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="startLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Start:     </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="startTime">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="endFrame">
        <property name="styleSheet">
         <string notr="true">color: rgb(186, 186, 186);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="endLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>End:      </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="endTime">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="totalFrame">
        <property name="styleSheet">
         <string notr="true">color: rgb(186, 186, 186);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="totalLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Total:    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="totalTime">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>64</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="btnsFrame">
        <property name="styleSheet">
         <string notr="true">QFrame{
	padding: 5px
}

QPushButton{
	color: rgb(179, 182, 186);
	border: 2px solid rgb(10, 10, 29);
	background-color: rgb(0, 0, 43);
	border-radius: 10px;
}

QPushButton::hover{
	color: rgb(143, 145, 148);
	background-color: rgb(0, 0, 85);
}

QPushButton::pressed{
	color: rgb(254, 254, 254);
	background-color: rgb(0, 0, 135);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>39</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveBtn">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(36, 131, 31);</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>38</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="discardBtn">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(135, 47, 13);</string>
           </property>
           <property name="text">
            <string>Discard</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>39</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
